src/AppBundle/Entity/Comment.php

```diff
@@ -91,11 +94,17 @@ public function isLegitComment()
         return !$containsInvalidCharacters;
     }
 
+    /**
+     * @return int
```

Comment:
@javiereguiluz Sorry, it's probably too late, but we don't need these `@return` statements for getters since we already have it on properties and PhpStorm could resolve them by itself: https://github.com/cafferata/symfony-demo/blob/e039e604a31754e371811b5dba2b0eb738af07fc/src/AppBundle/Entity/Comment.php#L33

But we need them for setters though.

Some discussion about it here: https://github.com/symfony/symfony-demo/pull/394